app.js

```diff
@@ -197,7 +259,12 @@ class SrsClient extends Emitter {
     try {
       this.uac = await this.srf.createUAC(this.srsUrl, {
         headers: {
-          'Content-Type': 'multipart/mixed;boundary=uniqueBoundary',
+          'Supported': 'replaces,resource-priority,sdp-anat',
+          'Allow': 'REGISTER,OPTIONS,INVITE,ACK,CANCEL,BYE,NOTIFY,PRACK,REFER,INFO,SUBSCRIBE,UPDATE',
+          'x-audc-call-id' : this.srsRecordingId,
```

Comment:
is this header (x-audc-call-id) there for all cases?  We would not want that.  Part of the customization should be to generate a headers object for the given "type" (ie smart tap)